kamon-play-2.5.x/src/main/scala/kamon/play/instrumentation/WSInstrumentation.scala

```diff
@@ -37,8 +41,36 @@ class WSInstrumentation {
       val segment = ctx.startSegment(segmentName, SegmentCategory.HttpClient, PlayExtension.SegmentLibraryName)
       val response = pjp.proceed().asInstanceOf[Future[WSResponse]]
 
-      response.onComplete(result ⇒ segment.finish())(SameThreadExecutionContext)
+      response.onComplete {
+        case Success(result) ⇒
+          PlayExtension.httpClientMetrics.recordResponse(segmentName, result.status.toString)
+          segment.finish()
+        case Failure(error) ⇒
+          segment.finishWithError(error)
+      }(SameThreadExecutionContext)
       response
     } getOrElse pjp.proceed()
   }
-}
\ No newline at end of file
+}
+
+/**
+ *  Counts HTTP response status codes into per status code and per trace name + status counters. If recording a HTTP
+ *  response with status 500 for the trace "http://localhost:9000/badoutside_200", the counter with name "500" as
+  *  well as the counter with name "http://localhost:9000/badoutside_500" will be incremented.
+ */
+class HttpClientMetrics(instrumentFactory: InstrumentFactory) extends GenericEntityRecorder(instrumentFactory) {
```

Comment:
I totally agree with that, but making that change in code now (it shouldn't be called `HttpServerMetrics` if it is now covering both server and client) would break other modules using it.. I would say, leave it as it is and we create an issue to track that for Kamon 0.7.0, wdyt?